src/tools/m65.c

```diff
@@ -1256,20 +1262,32 @@ char* test_states[16] = { "START", " SKIP", " PASS", " FAIL", "ERROR", "C#$05",
 char msgbuf[160];
 char testname[160];
 unsigned char inbuf[8192];
+unsigned int failcount;
+FILE* logPtr;
 
 void unit_test_log(unsigned char bytes[4])
 {
   int test_issue = bytes[0] + (bytes[1] << 8);
   int test_sub = bytes[2];
+  char outstring[255];
 
   // dump_bytes(0, "bytes", bytes, 4);
 
   // log test name if we have one
   if (testname[0]) {
     fprintf(stderr, "%s: ", testname);
+    if (logPtr) {
+      fprintf(logPtr, "%s: ", testname);
+    }
   }
 
-  fprintf(stderr, "%s (Issue#%d, Test #%d)\n", test_states[bytes[3] - 0xf0], test_issue, test_sub);
+  sprintf(outstring, "%s (Issue#%d, Test #%d)\n", test_states[bytes[3] - 0xf0], test_issue, test_sub);
```

Comment:
Again, this is fine for now, but it would be good for us to think ahead to having a format for these lines that is easy to parse, so that we can more easily make tools that can read and parse these log files. I'm not sure if there are any existing official test log file formats, e.g., from TAP, that we should emulate.

Reply:
Oh, yes, and please include a timestamp on these lines.